proposals/2023-01-regen-ledger-v5.0-upgrade/proposal.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Regen Ledger v5.0 Upgrade",
+  "description": "This is a software upgrade proposal for the upgrade to Regen Ledger v5.0. If passed, this proposal would commit Regen Mainnet to halting the application binary for Regen Ledger v4.1 at approximately 16:00 UTC on January 17th at block height 8942600 and requiring nodes to restart with the application binary using Regen Ledger v5.0. \n\nMore details can be found in the long form proposal: https://github.com/regen-network/governance/tree/main/proposals/2023-01-regen-ledger-v5.0-upgrade",
+  "upgrade-height": 8942600,
+  "upgrade-info": {
+    "binaries": {
+      "linux/amd64": "https://github.com/regen-network/regen-ledger/releases/download/v5.0.0/regen-ledger_5.0.0_linux_amd64.zip?checksum=sha256:edbf5beaa769f971cf6b6f3c5e45cc3f38ac7d5b9dd005cddc821ccc37771155",
+      "linux/arm64": "https://github.com/regen-network/regen-ledger/releases/download/v5.0.0/regen-ledger_5.0.0_linux_arm64.zip?checksum=sha256:30b5cc181a970d957577ccee712371c9e88f5cde511d2abdf4a185aa2c97ab1d",
+      "darwin/amd64": "https://github.com/regen-network/regen-ledger/releases/download/v5.0.0/regen-ledger_5.0.0_darwin_amd64.zip?checksum=sha256:XXX",
```

Comment:
cc @clevinson we need to push a darwin package build and update the checksums txt file.